enhancements/oc/mirroring-manifests.md

```diff
@@ -26,9 +29,18 @@ status: provisional|implementable
 
 ## Summary
 
-For disconnected clusters, `oc adm release mirror ...` already helps users copy the release image and its dependencies into their local mirror.
-This enhancment extends that command to also create and apply manifests to their local cluster, to bring in information that cannot be represented by container images.
-For example, `oc` might produce ConfigMap manifests containing the release image signature which the cluster-version operator could use to [verify the mirrored release][cvo-config-map-signatures].
+For disconnected clusters, `oc adm release mirror ...` already helps users copy the release image and its dependencies into their local mirror. As of 4.2 and 4.3, the following command:
+
+```console
+$ oc adm release mirror \
```

Comment:
Why are we promoting this up out of the *Proposal* section into the *Summary*?  It seems like an implementation detail that we don't need for the high-level summary.

Reply:
It helps explain the current impl and then I refer to the Summary in section "Pushing manifests to disk for later application" as part of explaining an ehancement need there

Reply:
But why do we need to show an example `oc` command to explain the current implementation, especially just for this single images-to-disk case?

Reply:
I do not see any harm showing the current implementation. At least it helps in better review of this PR.

Reply:
I'm not saying "don't show the current implementation", I'm saying "this is not summary-level information, this is proposal-level background".